### ISSUE TITLE
Modal Shortcode should output HTML in wp_footer

### ISSUE THREAD

**Issue by scharc** [filipstefansson/bootstrap-3-shortcodes#112]
I used css opacity on the parent element that contains the modal button... therefor the modal got the opacity inherited...

`jQuery('.modal').detach().appendTo('footer')`

was my workaround, but it would be nice to us the wp_footer-action hook to insert the HTML into the footer
